src/main/java/games/strategy/triplea/delegate/EndTurnDelegate.java

```diff
@@ -160,43 +160,48 @@ private static Territory getRandomTerritory(final Collection<Territory> territor
     return list.get(random);
   }
 
-  private String createResources(final IDelegateBridge bridge, final boolean negativeResources) {
-    final StringBuilder endTurnReport = new StringBuilder();
+  private String addUnitCreatedResources(final IDelegateBridge bridge) {
+
+    // Find total unit generated resources for all owned units
     final GameData data = getData();
     final PlayerID player = data.getSequence().getStep().getPlayerID();
-    final Match<Unit> myCreatorsMatch = Match.all(Matches.unitIsOwnedBy(player),
-        negativeResources ? Matches.UnitCreatesResourcesNegative : Matches.UnitCreatesResourcesPositive);
```

Comment:
`UnitCreatesResourcesNegative` and `UnitCreatesResourcesPositive` appear to no longer be referenced after this change.  Removing them would get rid of about 40 lines of code.